polaris/hub/client.py

```diff
@@ -797,114 +775,72 @@ def _upload_benchmark(
             benchmark_json["datasetArtifactId"] = benchmark.dataset.artifact_id
             benchmark_json["access"] = access
 
-            path_params = (
-                "/v1/benchmark" if artifact_type == ArtifactSubtype.STANDARD.value else "/v2/competition"
-            )
+            path_params = "/v1/benchmark"
             url = f"{path_params}/{benchmark.owner}/{benchmark.name}"
             response = self._base_request_to_hub(url=url, method="PUT", json=benchmark_json)
             response_data = response.json()
 
             progress_indicator.update_success_msg(
-                f"Your {artifact_type} benchmark has been successfully uploaded to the Hub. "
+                f"Your benchmark has been successfully uploaded to the Hub. "
                 f"View it here: {urljoin(self.settings.hub_url, url)}"
             )
             return response_data
 
     def get_competition(
-        self,
-        owner: str | HubOwner,
-        name: str,
-        verify_checksum: ChecksumStrategy = "verify_unless_zarr",
+        self, owner: str | HubOwner, name: SlugCompatibleStringType
```

Comment:
**Suggestion**
Start using the artifact identifier as is, since that's what users are going to be using.

```suggestion
        self, artifactId: str
```